cds/modules/records/mappings/records/videos/project/project-v1.0.0.json

```diff
@@ -3,6 +3,14 @@
     "project-v1.0.0": {
       "date_detection": true,
       "properties": {
+        "_created": {
+          "type": "date",
+          "copy_to": "created"
+        },
+        "_updated": {
+          "type": "date",
+          "copy_to": "updated"
```

Comment:
Can you delete the `copy_to` line from the schemas? It was my mistake, we do not need that.

Reply:
Sure, I will do it now.